code/API_definitions/network_access_management.yaml

```diff
@@ -0,0 +1,2863 @@
+openapi: 3.0.3
+
+info:
+  title: CPE Management API
+  version: 0.0.1
+  description: |
+    Service enabling API for managing network operator supplied network access devices. This APIs initial focus is on managing **Isolated Networks** and device reboot requests.
+
+    # Introduction
+    An isolated network is a network access configuration where each configuration is for a logically separated Local Area Network (LAN). Application developers can create, modify, or remove these networks which could be useful in scenarios such as:
+
+    - A property owner wants to create a temporary network for the duration of a short-term booking or long-term rental.
+    - A property manager wants to replicate the network access details across multiple network access devices at one or more locations such as common areas, offices, etc.
+    - A security/privacy-minded user wants to create one or more networks for their guests or Internet-of-Thing (IoT) network clients.
+
+    A reboot request is simply a request to reboot a network operator supplied network access device. Application developers can create these requests which could be useful in scenarios such as:
+
+    - A property owner wants to recover from a faulty network state by rebooting a gateway device.
+    - A property manager wants to reboot multiple devices at one or more locations such as common areas, offices, etc.
+
+    # Relevant Terms and Definitions

```

Comment:
Terms should be aligned with other CAMARA APIs and https://github.com/camaraproject/Commonalities/blob/main/documentation/Glossary.md

Reply:
Should we define "Customer Premise Equipment" (CPE)? Which in case of Home Devices QoD would be the Home Router...
Or Network Access Device if it replaces CPE term...

Reply:
CPE should get removed and updated with the new terminology.  Will make those updates.

Reply:
@jpengar Thanks for the feedback.  We've created #14 to track this issue and will resolve in another PR prior to the initial beta release.